api-specificatie/zrc/openapi.yaml

```diff
@@ -99,69 +531,39 @@ components:
           format: uri
           maxLength: 200
           minLength: 1
-        status:
-          title: Status
-          description: 'Indien geen status bekend is, dan is de waarde ''null'''
+        registratiedatum:
+          title: Registratiedatum
+          description: >-
+            De datum waarop de zaakbehandelende organisatie de ZAAK heeft
+            geregistreerd
           type: string
-          format: uri
-          readOnly: true
+          format: date
+        toelichting:
+          title: Toelichting
+          description: Een toelichting op de zaak.
+          type: string
+          maxLength: 1000
         omschrijving:
           title: Omschrijving
           description: Een korte omschrijving van de zaak.
           type: string
           maxLength: 80
-        # TODO: in het RGBZ 2.0 model loopt dit via de rollen/betrokkenen
-        # initiator:
-        #   title: Initiator
-        #   description: URL naar de initiator van de ZAAK.
-        #   type: string
-        #   format: url
-        registratiedatum:
-          title: Registratiedatum
-          description: De datum waarop de zaakbehandelende organisatie de ZAAK heeft geregistreerd
-          type: string
-          format: 'date'
-        # TODO: volgens RGBZ 2.0 komt kanaal uit KLANTCONTACT
-        # kanaal:
-        #   description: Kanaal via welke de zaak is gemeld
-        #   type: string
-        zaakgeometrie:  # NOTE: komt ook voor bij OBJECT
+        zaakgeometrie:
           title: Zaakgeometrie
-          description: Punt, lijn of multi-vlak geometrie-informatie, in WKT formaat.
-          type: string
+          description: 'Punt, lijn of (multi-)vlak geometrie-informatie, in GeoJSON.'
+          type: object
         objecten:
           title: OBJECTen
           description: Lijst met URL's naar aan de zaak gerelateerde objecten
           type: array
           items:
             type: string
```

Comment:
Na "type: string", zou je "format: uri" verwachten omdat het hier gaat om een URL naar het gerelateerde zaakobject

Reply:
@zomwoof klopt, al werd dit nog niet gebruikt en kwam het uit de eerste opzet van @andyverberne . In https://github.com/VNG-Realisatie/gemma-zaken/pull/200/files#diff-c3c469d285e9346c220f1e67491af11cL132 is dit dan ook weggehaald tot uit een US blijkt dat dit wel meteen moet meekomen, en dan doen we 't goed :)